tests/test_base.hpp

```diff
@@ -220,6 +220,7 @@ class server_config
 
 	config_data options;
 	std::vector<config_data> backends;
+	std::unordered_set<size_t> custom_remote_indexes; // if not set, server'll interconnect with all other servers
```

Comment:
It should be better described what effect does it have.

Reply:
Done.